### ISSUE TITLE
VPAT 1.1.1 Alt-Text

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#21]
Screengrab/videos (insert file names here)
1, Initial Screen.png

Link
https://permit-finder.digital-staging.boston.gov

Environment
(Device > OS > Browser)
Computer > macOS 10.14.6 > Safari 12.1.2

Brief Description
Some Images need additional description.

Steps to reproduce
1. See the image.  The City of Boston seal is labeled as the seal, but does not indicate the link destination.  Other images are also highlighted that should have more description search text.

Desired Result
All images have description alt-text.

Actual Result
Some alt-text is missing or additional descriptions are needed.
![1, Initial Screen](https://user-images.githubusercontent.com/33845414/62467185-20b5b200-b761-11e9-809b-c32ef1e0a7dd.png)

**Comment by paylakatel** [comment]
This looks good on staging - we can move to production.

**Comment by phillipbentonkelly** [comment]
Deployed, Live; Leaving the ticket un-closed until Monday
